resources/js/components/Controller.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import axios from "axios";
+
+class Controller extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            trigger: 0,
+        }
+    }
+
+    render() {
+        return (
+            <>
+                <p><u>SYNTH CONTROLS</u>:</p>
+                {/* TIGGER TONE */}
```

Comment:
I like what you're doing here. It's pretty slick, and is the basis for some cool functionality.